cognite/power/data_classes.py

```diff
@@ -412,6 +412,33 @@ def current_limits_overview(self) -> "pd.DataFrame":
             for coeff in temp_coefficients
         ).set_index(["name", "type", "temperature"])
 
+    def load_duration_curve(
+        self,
+        start,
+        end="now",
+        terminal=1,
```

Comment:
What is the meaning of the terminal numbers?

Reply:
for line a-b, the sensor at the 'a' end is terminal 1, and at the 'b' end terminal 2, afaik